src/sharing/components/ShareAutocomplete.jsx

```diff
@@ -23,39 +23,70 @@ export default class ShareAutocomplete extends Component {
     )
   }
 
-  onSuggestionsFetchRequested ({ value }) {
-    this.setState(state => ({
-      ...state,
-      suggestions: this.computeSuggestions(value)
-    }))
+  onSuggestionsFetchRequested = ({ value }) => {
+    this.setState(state => ({...state, suggestions: this.computeSuggestions(value)}))
+  }
+
+  onSuggestionsClearRequested = () => {
+    this.setState(state => ({...state, suggestions: []}))
+  }
+
+  onChange = (event, { newValue, method }) => {
+    // A suggestion has been picked
+    if (typeof newValue === 'object') {
+      this.onPick(newValue)
+    } else {
+      this.setState(state => ({...state, inputValue: newValue}))
+    }
+  }
+
+  onKeyPress = (event) => {
+    // The user wants to add an unknown email
+    if (event.key === 'Enter' && this.state.inputValue !== '') {
```

Comment:
It look like `event.key` is [not supported in safari](https://developer.mozilla.org/fr/docs/Web/API/KeyboardEvent/key), so it's probably safer to use the keycode instead.

Reply:
You're right, fixed.